tests/protocols/test_calibrationmodel.py

```diff
@@ -143,18 +144,18 @@ def fit_si(spe_sil):
         profile="Pearson4", candidates=cand, no_fit=False, vary_baseline=False
     )
     df = fitres.to_dataframe_peaks().sort_values(by="height", ascending=False)
-    difference = abs(df.iloc[0]["center"] - si_peak)
-    assert difference < 1e-2, f"Si peak found at {df.iloc[0]['center']}"
+    difference = abs(df.iloc[0]["position"] - si_peak)
```

Comment:
Even though `position` provides better results due to the 'Person4Model' problems, `center` sounds the right thing to me as it is the mean of the multiple lines (in the multi-line understanding of the asymmetric silicon peak). `position` will show the most prominent one. In case the multiple lines have similar intensities the `position` parameter becomes unstable.
(Just wanted to share my opinion and concerns, and do not insist reverting to `center`)

Reply:
I strongly disagree. We are looking for the peak - and mode is by definition the peak . Mode is different than mean for skewed distributions.  (of course they are the same in symmetric distributions like Gaussian)

Reply:
It is not a question about general considerations. In fact it really depends what stands behind the 520.xx nm number. I haven't seen a definitive answer to this yet. And as in physics the peaks are commonly defined by their centroids (`center` in these terms), i wanted to draw attention here.
No need to explain me what are mode and mean of a statistical distribution.

Reply:
it's a public discussion,  hence this is an argument for the readers why this decision , not a personal argument.
we _are_ fitting a distribution, and looking for the mode.  in fact it would be more consistent if all peak profiles return both mode and mean, even if it is the same (not just for Pearson4).

Reply:
The thing is that i am not convinced by the materials i found that $520 \mathrm{cm}^{-1}$ corresponds to the mode. I more tend to believe that it corresponds to the mean or center-of-gravity, because this is what we usually mean by position of a peak.